terraform/rackspace/modules/etcd_discovery/main.tf

```diff
@@ -0,0 +1,23 @@
+# Variables
```

Comment:
seems like this can be moved to a more top-level which cuts across all providers (and we can re-use in other providers) - thoughts? can we create a follow-up ticket to address this?